modules/ROOT/partials/configuration/plugins.adoc

```diff
@@ -15,6 +20,14 @@ tinymce.init({
 });
 ----
 
-NOTE: Each plugin entry should be separated by a blank space. Alternatively, an array of plugin names can be provided.
+=== Example: Using `+plugins+` with an array
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins : [ 'advlist', 'autolink', 'link', 'image', 'lists', 'charmap', 'preview' ]
```

Comment:
Looks like the above snippet has this as well
```suggestion
  plugins: [ 'advlist', 'autolink', 'link', 'image', 'lists', 'charmap', 'preview' ]
```

Reply:
Fixed